database/src/migrations/20240308110000_survey_progress.ts

```diff
@@ -0,0 +1,93 @@
+import { Knex } from 'knex';
+
+/**
+ * Create new tables with initial seed data:
+ * - survey_progress
+ *
+ * Update existing tables:
+ * - Add 'progress' column to Survey table
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`--sql
+    ----------------------------------------------------------------------------------------
+    -- Create survey progress lookup table
+    ----------------------------------------------------------------------------------------
+    SET search_path = biohub;
+
+    CREATE TABLE survey_progress (
+      survey_progress_id                            integer            GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+      name                                          varchar(32)        NOT NULL,
+      description                                   varchar(128),
+      record_end_date                               timestamptz(6),
```

Comment:
Technically optional since we don't really use it, but if you want to maintain consistency with the other tables, we would want to add the `record_effective_date` column.

Reply:
Edit: we do seem to reference it in a few places, just not a lot